kernel/kernel-impl/src/main/java/org/sakaiproject/content/impl/CollectionAccessFormatter.java

```diff
@@ -130,158 +125,49 @@ public void format(ContentCollection x, Reference ref, HttpServletRequest req, H
 					new Object[]{ formattedText.escapeHtml(pl.getProperty(ResourceProperties.PROP_DISPLAY_NAME))}) + "</title>");
 			out.println("<link href=\"" + webappRoot + skinRepo+ "/" + skinName + 
 			"/access.css\" type=\"text/css\" rel=\"stylesheet\" media=\"screen\">");
+			//we need tool.css to load font-awesome
+			out.println("<link href=\"" + webappRoot + skinRepo+ "/" + skinName + 
+			"/tool.css\" type=\"text/css\" rel=\"stylesheet\" media=\"screen\">");
 			out.println("<script type=\"text/javascript\" language=\"JavaScript\" src=\"/library/js/headscripts.js\"></script>");
 			out.println("<script type=\"text/javascript\">includeLatestJQuery(\"access\");</script>");
+			
+			//folder-listing libraries (JS + CSS) 
+			out.println("<script type=\"text/javascript\" language=\"JavaScript\" src=\"/library/editor/ckextraplugins/folder-listing/js/file-tree.js\"></script>");
```

Comment:
If these are going to be used more widely than just in the ckeditor should move them somewhere better?